src/cleanvision/imagelab.py

```diff
@@ -226,26 +365,26 @@ def report(
         issue_summary = self.issue_summary[
             self.issue_summary["issue_type"].isin(computed_issue_types)
         ]
-        self.print_issue_summary(issue_summary)
+        self._pprint_issue_summary(issue_summary)
 
         self.visualize(
             issue_types=computed_issue_types,
             examples_per_issue=report_args["examples_per_issue"],
         )
 
-    def print_issue_summary(self, issue_summary: pd.DataFrame) -> None:
+    def _pprint_issue_summary(self, issue_summary: pd.DataFrame) -> None:
```

Comment:
Typo when renaming this method?

```suggestion
    def _print_issue_summary(self, issue_summary: pd.DataFrame) -> None:
```

Reply:
My intention was to use pp similar to pretty print, but either one works

Reply:
Ohh I see, then feel free to ignore my comment.